src/azure-cli/azure/cli/command_modules/resource/tests/latest/crosstenant_vm_deploy.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$schema": "https: //schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualMachines_veerCrossTenantVM_2_name": {
+            "defaultValue": "veerCrossTenantVM-2",
+            "type": "String"
+        },
+        "SIG_ImageVersion_id": {
+            "type": "String"
+        },
+        "NIC_id": {
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "apiVersion": "2019-07-01",
+            "type": "Microsoft.Compute/virtualMachines",
+            "name": "[parameters('virtualMachines_veerCrossTenantVM_2_name')]",
+            "location": "eastus",
+            "tags": {},
+            "properties": {
+                "hardwareProfile": {
+                    "vmSize": "Standard_DS1_v2"
+                },
+                "networkProfile": {
+                    "networkInterfaces": [
+                        {
+                            "id": "[parameters('NIC_id')]"
+                        }
+                    ]
+                },
+                "storageProfile": {
+                    "osDisk": {
+                        "createOption": "fromImage",
+                        "name": None,
```

Comment:
None is highlighted RED. Is it an error which Github can't stand?

Reply:
For JSON format, None value is not supported but it doesn't matter in ARM deployment. But I can give it valid value for better description and validity.